_news-items/2025-05-21-proofcraft.html

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2025 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Thank you Proofcraft, Silver sponsor of the seL4 Summit 2025
+anchor: sponsor25-proofcraft
+
+img: /Foundation/Membership/LOGOS/proofcraft.svg
+alt: Proofcraft Logo
+link: https://proofcraft.systems
+---
+
+<p>
+  The seL4 Foundation thanks <a href="https://proofcraft.systems">Proofcraft</a>
+  for becoming a Silver sponsor of the <a href="../Summit/2025">seL4
+  Summit 2025</a>.
+</p>
+<p>
+  Founded by the seL4 verification leaders, <a
+  href="https://proofcraft.systems">Proofcraft</a> offers commercial support,
+  verification projects, training and consulting on formal verification in
+  general, and involving seL4 specifically. By applying mathematical
```

Comment:
Please change to : "offers commercial support and projects in formal verification in general, and involving seL4 specifically."